api/ca.proto

```diff
@@ -55,3 +60,9 @@ message GetRootCACertificateRequest {}
 message GetRootCACertificateResponse {
 	bytes certificate = 1;
 }
+
+message GetUnlockKeyRequest {}
+
+message GetUnlockKeyResponse {
+	bytes unlock_key = 1;
```

Comment:
There are; but 256 is enough. Barring an attack against a specific CSPRNG (which almost certainly would be classical in nature), the general quantum attack is to treat the output side of the CSPRNG as an idealized PRF (this is a standard assumption), then query it using Grover's algorithm. A classical attacker experiences 256 bits of entropy (that is; they need 2**256 time and constant space to exhaust the input space). A quantum attacker uses Grover's algorithm to query the black box, and experiences 128 bits of entropy. 128 bits are considered enough forever.